### PR TITLE
Add payment strategy to carriers API and CSV.

### PR DESCRIPTION
**Pull request by r38y**

### CODE REVIEW

File: brokers/csv.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Required fields are **bold**.
 * dot_number
 * days_to_pay
 * quickpay - true or false, default is false
+* payment_strategy - defaults to direct, options are direct or triumphpay
```
Comment (kaleposhobios) [inline brokers/csv.md:49]: I think we should add more information about what this means, if at all possible. Like...idk. "Helps automatic checks of remit-to address" or something?

**Review by kaleposhobios — APPROVED** [review]

File: brokers/csv.md (hunk 1 above)
Comment (r38y) [reply brokers/csv.md:49]: Will it do that?

Comment (kaleposhobios) [reply brokers/csv.md:49]: Not yet. But without anything how will consumers know what to pass? I guess maybe just an expanded defintion: "Describes who is responsible for paying the carrier -- direct means you'll manage the payments (amount, remit-to address) and some other value means that 3rd party is responsible for those details and you don't manage them."

Maybe?

Comment (r38y) [reply brokers/csv.md:49]: Right now it doesn't do anything, the customer just needs to know how to send it. Maybe once we do something with it we can update it to say what it actually does?

**Review by kaleposhobios — APPROVED** [review]

**Pull request merged**